build_torch_xla_libs.sh

```diff
@@ -54,9 +54,9 @@ if [[ ! -z "$BAZEL_JOBS" ]]; then
   MAX_JOBS="--jobs=$BAZEL_JOBS"
 fi
 
-OPTS+=(--cxxopt="-std=c++14")
+OPTS+=(--action_env=BAZEL_CXXOPTS="-std=c++17")
```

Comment:
Does passing `--cxxopt=-"std=c++17"` instead work, too?

Reply:
Yes, that works. This format seems to be preferred according to a blog post I read at the time. Don't have the reference handy unfortunately.